pkg/config/flags.go

```diff
@@ -0,0 +1,32 @@
+package config
+
+import (
+	"flag"
+	"time"
+
+	"github.com/openshift/elasticsearch-proxy/pkg/util"
+)
+
+func newFlagSet() *flag.FlagSet {
+	flagSet := flag.NewFlagSet("elasticsearch-proxy", flag.ExitOnError)
+
+	flagSet.String("https-address", ":8443", "<addr>:<port> to listen on for HTTPS clients")
+
+	flagSet.String("tls-cert", "", "path to certificate file")
```

Comment:
to be clear, these are for the proxy to use for incoming requests?

Reply:
Used in `pkg/proxy/http.go:` to setup listening for `client->proxy`